### ISSUE TITLE
TypeScript types not working due to typo in package.json

### ISSUE THREAD

**Issue by alundeen** [googleapis/nodejs-storage#362]
Hello,

Thanks for releasing a TypeScript version! I'm excited to have a typed interface available.

However, the `types` field in package.json currently references a non-existent file as of the 2.0.0 release:

```
  "types": "./build/src/index.t.ds",
```

I believe the correct file is:

```
  "types": "./build/src/index.d.ts",
```

(Note the swapped letters in the file extension).

This causes an error when trying to import the module in a TypeScript project: `TS7016: Could not find a declaration file for module '@google-cloud/storage'.`

Thanks!

**Comment by JustinBeckwith** [comment]
🤦‍♂️

**Comment by alundeen** [comment]
Thanks for the quick response!

**Comment by jakubdrabik** [comment]
@JustinBeckwith Hi, thanks for the quick fix, when shall we expect a new patched version on NPM?

**Comment by welkie** [comment]
@jakubdrabik This fix is on NPM now with `2.0.1`.
